continue-reading.html

```diff
@@ -1,13 +1,155 @@
 <!DOCTYPE HTML>
+<link href="continuereading.css" rel="stylesheet" />
+<link href="https://fonts.googleapis.com/css2?family=Peddana&display=swap" rel="stylesheet">
+
 <html>
-  <head>
-    <title>Continue Reading</title>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
-  </head>
-  <body>
-    <section id="header"></section>
-    <section id="main"></section>
-    <section id="footer"></section>
-  </body>
-</html>
\ No newline at end of file
+
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
+</head>
+
+<body>
+  <!--Content Start and Title...-->
+  <a href="index.html" class="continuetitle"> Let's Read More About This!</a>
+  <h2>I guess If You Really Want Too...</h2>
+  <hr class="linebrk">
+  <hr class="linebrk1">
+  <!--Above picture text and first picture/first screenshot of continue page-->
+  <section class="pagestart">
+    <h1 class="largetext"> There's more stuff down here</h1>
+    <h3>LOL</h3>
+    <img class="firstimage" img
+      src="https://bloximages.chicago2.vip.townnews.com/missoulian.com/content/tncms/assets/v3/editorial/1/ad/1adf1952-b067-5841-b4ff-043c6323c67e/585d6c846cf02.image.jpg?resize=1200%2C872">
+    <!--under image text here-->
+    <div class="underimagetext">Don't Tip Over!</div>
+    <div class="underimageparagraph">
+      &nbsp; &nbsp; Taxidermy velit kale chips non unicorn hammock consequat. Lorem crucifix sunt meditation organic.

```

Comment:
Again, `&nbsp` is not the best way to go about creating the styling for this page